cmd/tailscale/cli/debug.go

```diff
@@ -784,6 +790,29 @@ func runDaemonLogs(ctx context.Context, args []string) error {
 	}
 }
 
+func runDaemonBusEvents(ctx context.Context, args []string) error {
+	logs, err := localClient.TailBusEvents(ctx)
+	if err != nil {
+		return err
+	}
+	d := json.NewDecoder(logs)
+	for {
+		var line struct {
+			Count int      `json:"Count"`
+			Type  string   `json:"Type"`
+			From  string   `json:"From"`
+			To    []string `json:"To"`
+			Event any      `json:"Event"`
+		}
```

Comment:
These are the default names anyway, so we can skip the tags:

```suggestion
		var line struct {
			Count int
			Type  string   
			From  string
			To    []string
			Event any      
		}
```

(I don't think we need `omitzero` or `omitempty` here, since we expect these to be populated in the ordinary course, and even an empty `To` field is meaningful as such)